resources/[gameplay]/helpmenu/admins.lua

```diff
@@ -4,7 +4,7 @@ admins = {
 	{ name = "Kreator", title = "Server Manager", accountName = "Goldberg" },
 	{ name = "Mihoje", title = "Developer", accountName = "Mihoje" },
 	{ name = "Haxardous", title = "Developer", accountName = "Axa" },
-    { name = "Nick_026", title = "Gayloper", accountName= "Nick_026" }
+    { name = "Nick_026", title = "Gayloper", accountName= "Nick_026" },
```

Comment:
accountName= "Nick_026"
should be
accountName = "Nick_026"

Reply:
Done